tpp/src/main/AccessTokenRequestBuilder.js

```diff
@@ -8,4 +9,38 @@ export default class AccessTokenRequestBuilder extends TokenRequestBuilder {
     constructor(payload: Object) {
         super(payload);
     }
+
+    /**
+     * Optional. Sets the source account to bypass account selection.
+     *
+     * @param source
+     * @return AccessTokenRequestBuilder
+     */
+    setSource(source: Object): AccessTokenRequestBuilder {
+        if(!this.requestPayload.accessBody.resourceTypeList)
+        {
+            this.requestPayload.accessBody.resourceTypeList = {};
+        }
+        this.requestPayload.accessBody.resourceTypeList.source = source;
+        return this;
+    }
+
+    /**
+     * Adds a transfer destination to a access token request.
+     *
+     * @param destination
+     * @return AccessTokenRequestBuilder
+     */
+    addTransferDestination(destination: TransferDestination): AccessTokenRequestBuilder {
```

Comment:
Can you pls double-check? Afaik transfer destination doesn't exist in AIS flow

Reply:
Updated.
We will only use setSource to add all the mandatory fields.
mandatoryFields -> access -> fields
![Screenshot 2021-12-16 at 3 21 20 PM](https://user-images.githubusercontent.com/78541086/146348825-e50c5b2e-5e45-43ac-a4ff-413f78f56038.png)